### DISCUSSION TITLE
How to config VSCode extension to use binary from specific path instead of packaged one in extension?

### DISCUSSION THREAD

**Discussion by mgzenitech** [microsoft/pyright#8283] [Q&A]
We install all our linters,formatters,etc. as cli tools so that people using VSCode and CLI would be able to run same version, same config of tools. We've installed pyright cli to `.venv/bin/pyright`. How to force VSCode extension to use this one instead of packaged one in extension?

P.S. Python path is already configured properly for .venv:
```
"python.defaultInterpreterPath": "${workspaceFolder}/.venv/bin/python"
```

**Comment by erictraut** [comment]
The VSCode extension includes own copy of pyright. It's not able to use a different binary.

**Reply by mgzenitech** [reply]
Would it be possible to expose this configuration as it is done in other extensions?
for example few:
```json
"yapf.importStrategy": "fromEnvironment",
"ruff.importStrategy": "fromEnvironment"
```
?

**Reply by erictraut** [reply]
That would require a completely different architecture/design, so we have no plans to do this.
